grpc-server/template_test.go

```diff
@@ -50,7 +67,26 @@ func TestCreateTemplate(t *testing.T) {
 	}{
 		"SuccessfullTemplateCreation": {
 			args: args{
-				db:        mock.DB{},
+				db: mock.DB{
+					CreateTemplateFunc: func(ctx context.Context, name string, data string, id uuid.UUID) error {
+						if len(templateDB) > 0 {
```

Comment:
isn't len(templateDB) == 0 always because `clearTemplateDB` is being called?

Reply:
Yes, we can remove that check in the test case where only one template is being created.